schema/enums/StakeholderStatusType.schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/enums/StakeholderStatusType.schema.json",
+  "title": "Enum - Stakeholder Status Type",
+  "description": "Enumeration of types of activity statuses for a stakeholder",
+  "type": "string",
+  "enum": [
+    "ACTIVE",
+    "LEAVE_OF_ABSENCE",
+    "TERMINATION_VOLUNTARY_OTHER",
+    "TERMINATION_VOLUNTARY_GOOD_CAUSE",
+    "TERMINATION_VOLUNTARY_RETIREMENT",
+    "TERMINATION_INVOLUNTARY_OTHER",
+    "TERMINATION_INVOLUNTARY_DEATH",
+    "TERMINATION_INVOLUNTARY_DISABILITY",
+    "TERMINATION_INVOLUNTARY_WITH_CAUSE"
+  ],
+  "$comment": "Copyright © 2024 Open Cap Table Coalition (https://opencaptablecoalition.com) / Original File: https://github.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/tree/main/schema/enums/StakeholderStatusType.schema.json"
+}
```

Comment:
@jacobyavis  Is there logic that we foresee driven by the `StakeholderStatusType` (specifically the various inactive reasons) or do we see this as additional metadata to explain the reason why a stakeholder is active/inactive. If it's the latter, what are your thoughts if an enum with the values of `ACTIVE` and `INACTIVE` and an accompanying `StatusReason` of type string in the `Stakeholder` schema could accomplish the same task while allowing for more flexibility.

Reply:
That makes sense to me, thanks for the context!